packages/cactus-plugin-keychain-memory/package.json

```diff
@@ -79,12 +84,17 @@
   "homepage": "https://github.com/hyperledger/cactus#readme",
   "dependencies": {
     "@hyperledger/cactus-common": "0.3.0",
+    "@hyperledger/cactus-core": "0.3.0",
     "@hyperledger/cactus-core-api": "0.3.0",
+    "express": "4.17.1",
     "joi": "14.3.1",
+    "prom-client": "^13.1.0",
```

Comment:
@jagpreetsinghsasan My usual thing about having exact versions specified for dependencies. :-)